src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,15 @@
-export const TodoList = () => {};
+import { TodoInfo } from '../TodoInfo';
+import { Todo, User } from '../../types/types';
+
+type Props = {
+  todos: Todo[];
+  users: User[];
+};
+
+export const TodoList = ({ todos, users }: Props) => {
+  return (
+    <section className="TodoList">
+      <TodoInfo todos={todos} users={users} />
```

Comment:
looks strange, you shoud map the todos array here and save in TodoInfo only todo information
```
{todos.map(todo => (
   <TodoInfo todo={todo} users={users}/>
))}
```

Reply:
ok, thx)